components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -366,51 +372,56 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
     }
 
     async ensureWorkspaceAuth(instanceID: string, retry: boolean) {
```

Comment:
Is it possible to have any telemetry on what's retry count for this? We may be able to adjust things to better match the average case if we had that data.

Reply:
I don't think we have a concept for consistent tracking of retries/errors in the frontend, yet.

Adding a TODO for now in case we ever get to it!